xblock/plugin.py

```diff
@@ -41,6 +41,32 @@ def default_select(identifier, all_entry_points):  # pylint: disable=inconsisten
         raise AmbiguousPluginError(all_entry_points)
 
 
+def select_with_overrides(identifier, all_entry_points):
+    """
+    Honors the ability for an XBlock to override the default entry point.
+    Raise an exception when we have ambiguous entry points.
+    """
+
+    # Split entry points into overrides and non-overrides
+    overrides = []
+    block_entry_points = []
+
+    for block_entry_point in all_entry_points:
+        if block_entry_point.group.endswith('overrides'):
```

Comment:
```suggestion
        if block_entry_point.group.endswith('.overrides'):
```

super-nit: we're appending '.overrides' so we may as well check for that same exact suffix